### PR TITLE
add Expat

### PR DESCRIPTION
**Pull request by diedyesterdaywashere**
### SteamDB app page links to a few games using this

https://steamdb.info/app/1091500/info/

### Brief explanation of the change

added a rule for Expat

### CODE REVIEW

File: rules.ini [hunk 1]
```diff
@@ -193,6 +193,7 @@ cURL = curl(?:module|lib|d|-4|-ttv|-x64|64|_pluginw64_release)?\.(?:dll|exe|lib|
 DirectStorage = (?:^|/)dstorage\.dll$
 Discord = (?:^|/)(?:lib)?discord(?:|-rpc|_game_sdk)\.(?:dll|dylib|so)$
 EpicOnlineServices = (?:^|/)(?:lib)?eossdk
+Expat = (?:^|/)libexpat(?:w)?\.dll
```
Comment (Lifeismana) [inline rules.ini:196]:
```suggestion
Expat = (?:^|/)(?:lib|py)?expat
```
we can also look for the python lib

File: descriptions/SDK.Expat.md [hunk 2]
```diff
@@ -0,0 +1 @@
+[**Expat**](https://libexpat.github.io/) is a stream-oriented XML parser.
```
Comment (Lifeismana) [inline descriptions/SDK.Expat.md:1]:
```suggestion
[**Expat**](https://libexpat.github.io/) is a stream-oriented XML parser written in C.
```
Their full description is fine

**Review by Lifeismana — CHANGES_REQUESTED** [review]
Can you also add these 2 strings and some non matching tests
```
lib/python3.9/xml/parsers/expat.pyc
panda3d/spacebullet_1.9/linux_i386/pyexpat.so
```

File: descriptions/SDK.Expat.md (hunk 2 above)
Comment (diedyesterdaywashere) [reply descriptions/SDK.Expat.md:1]: didn't see the written in C part on their github repo readme (thats where i took it from) but yeah, wouldn't hurt i guess

**Comment by Lifeismana** [comment]
need to run the script

**Review by Lifeismana — APPROVED** [review]

**Comment by Lifeismana** [comment]
merge reverted via a force push

**Pull request merged**